aptos-move/aptos-aggregator/src/aggregator_extension.rs

```diff
@@ -465,11 +494,13 @@ mod test {
         aggregator_data.create_new_aggregator(aggregator_id_for_test(200), 200);
 
         // +0 to -601 is impossible!
-        let aggregator = aggregator_data.get_aggregator(aggregator_id_for_test(600), 600);
+        let aggregator =
+            aggregator_data.get_aggregator(aggregator_id_for_test(600), 600, &*TEST_RESOLVER, true);
         assert_err!(aggregator.sub(601));
 
         // Similarly, we cannot subtract anything from 0...
-        let aggregator = aggregator_data.get_aggregator(aggregator_id_for_test(200), 200);
+        let aggregator =
+            aggregator_data.get_aggregator(aggregator_id_for_test(200), 200, &*TEST_RESOLVER, true);
```

Comment:
Can we also add a test with false to test the new logic? I suppose test resolver can be made to work w. it.

Reply:
@vusirikala `TEST_RESOLVER` already uses `FakeDataStore` so you can just make sure it has some data before the test. I think in aggregator natives there are some tests which do that.

Reply:
I added 3 test cases with `aggregator_enabled` flag set to false

Reply:
Left some comments for tests